kube/minikube/deployments/scheduler-docker-deployment.yaml

```diff
@@ -0,0 +1,77 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
```

Comment:
I thought we said we shouldn't be running this one in Kubernetes?

Reply:
Yes, I'll drop this.